lib/kb_SPAdes/kb_SPAdesImpl.py

```diff
@@ -267,18 +270,58 @@ def make_ref(self, object_info):
         return str(object_info[6]) + '/' + str(object_info[0]) + \
             '/' + str(object_info[4])
 
+    def determine_unknown_phreds(self, reads,
+                                 phred64_reads,
+                                 phred33_reads,
+                                 unknown_phred_reads,
+                                 reftoname):
+        print("IN UNKNOWN CHECKING")
+        eautils = kb_ea_utils(self.callbackURL)
+        for ref in unknown_phred_reads:
+            rds = reads[ref]
+            obj_name = reftoname[ref]
+            files_to_check = []
+            f = rds['files']
+            if f['type'] == 'interleaved':
+                files_to_check.append(f['fwd'])
+            elif f['type'] == 'paired':
+                files_to_check.append(f['fwd'])
+                files_to_check.append(f['rev'])
+            print("FILES TO CHECK:" + str(files_to_check))
+            for file_path in files_to_check:
+                ea_stats_dict = eautils.calculate_fastq_stats({'read_library_path': file_path})
+                print("EA UTILS STATS : " + str(ea_stats_dict))
+                if ea_stats_dict['phred_type'] == '33':
```

Comment:
Is it known that kb_ea_utils returns always either '33' or '64' as phred type? What if it returns something unexpected? Then we may lose input data silently, right?